### DISCUSSION TITLE
Issues closed as completed can't be closed as not planned without reopening them

### DISCUSSION THREAD

**Discussion by carlosmintfan** [community/community#110447] [Mobile] [answered]
See heading
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.150.0 (123387055)
Device: iPhone 8 (iOS 16.4.1)
</details>

**Comment by stevepopovich** [comment]
Hi @carlosmintfan hmm I believe this is expected behavior. This is how the mobile apps and the website behaves. What behavior are you expecting here?

**Comment by ghost** [comment]
> See heading
> 
> ℹ️ Additional Debug Info
> GitHub for iOS, Version 1.150.0 (123387055) Device: iPhone 8 (iOS 16.4.1)

Yeah I too faced this one time

**Comment by carlosmintfan** [comment]
@stevepopovich The mobile app actually doesn't behave the same as the website. I'm expecting it to behave the same:

Website:
![grafik](https://github.com/community/community/assets/118076740/1cd28844-af59-43f3-9fc7-eb6ca113dc7e)
When an issue is open, you can either close it as "completed" or as "not planned". When you accidentally close it as completed, you can close it as "not planned" without reopening it and vice versa. However, while the mobile app also allows me to close an issue either as completed or not planned, after closing it only gives me the option to reopen it, not to close it the other method.

**Reply by stevepopovich** [reply, marked as answer]
Oh, you are totally right. That's fair. I will file this as a feature request with the team. Thanks for writing in.
